scripts/copy-files.js

```diff
@@ -0,0 +1,179 @@
+// Used by @mui/x-pickers and @mui/x-pickers-pro
```

Comment:
Could you rename this file to `copyFiles.js`.

Reply:
It is `copy-files` in the core but sure